packages/preview/uo-lth-msc-cs-thesis/0.1.0/template/main.typ

```diff
@@ -0,0 +1,116 @@
+#import "lib.typ": template, createAppendices
```

Comment:
This import (and any other import of `lib.typ` in the template subdirectory) should use the full `@preview/…` import.

Reply:
Got it! Thanks 🙏